lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -623,11 +626,15 @@ const handler = async function handler(event, context) {
         );
         return;
       }
+      
+      let tempCallData = await getCallDataFromChimeEvents(event);
+      callData = await getCallDataFromDdb(tempCallData.callId);
```

Comment:
wouldn't the callData.callId here still be the VC callid and not the modified callId that is needed for the KDS event?